### PR TITLE
wiki setup changes

### PR DESCRIPTION
**Pull request by onebeastchris**
Hey! As Jens asked for help with a small rewrite, i gave it a shot - here are my proposed changes.

Overview:
- split apart the standalone, and plugin instructions to reduce confusion
- added some config steps with the plugin setup 
- added the secureprofiles notice in the Prerequesites section
- made the "overview" a bit clearer - i like lists
- added more detail to the Bungee/Velocity setup 
- didn't change standalone instructions.
- added a quick tip to the pterodactly egg notice in the faq, as advised on discord https://discord.com/channels/613163671870242838/613168464634576897/1022433712983523368

This is my first time editing the wiki - the []() links may not work correctly.

### CODE REVIEW

File: _docs/geyser/setup.md [hunk 1]
```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
...
 - If you are connecting to an online mode Java server, a paid Java account is required. If you are running the server, you can bypass this requirement for your server with [Floodgate](/floodgate/).
 - Your Bedrock client has to be a supported version - at this time that is Bedrock version(s) {{ site.data.versions.bedrock }}.
 - If you are running the server, you need to have a UDP port opened. See below for more instructions.
+- Currently, Geyser does not support key signing. To disable it (on servers running version 1.19 and higher), follow these instructions: Spigot, Paper, & all forks: Set `enfore-secure-profile: false` in the `server.properties` file. On BungeeCord: Set `enforce_secure_profile: false` in your `config.yml`. Using Velocity: set `force-key-authentication = false` in your `velocity.toml`.
```
Comment (Camotoy) [inline _docs/geyser/setup.md:31]: I would elaborate that this is about chat signing.

File: _docs/geyser/faq.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ This is a Bedrock client issue caused by Bedrock being 32-bit and Java being 64-
 No, this is a translation tool to allow Bedrock players to connect to Java servers. It does not work the opposite way.
 
 ## Can I use Geyser with Pterodactyl Panel?
-Yes, you can use Geyser as a plugin or a mod for your Minecraft server software running on Pterodactyl. There is also an official egg for the standalone version. It supports auto-updating and has all config options easily editable. You can find it [here](https://github.com/GeyserMC/pterodactyl-stuff), download the JSON egg, and import it into your panel.
+Yes, you can use Geyser as a plugin or a mod for your Minecraft server software running on Pterodactyl. There is also an official egg for the standalone version. It supports auto-updating and has all config options easily editable. You can find it [here](https://github.com/GeyserMC/pterodactyl-stuff), download the JSON egg, and import it into your panel. Make sure the Geyser port (using UDP) is exported out to the host. If you're using Pterodactyl panel then you need to make sure you have allocated the port to the server.
```
Comment (Camotoy) [inline _docs/geyser/faq.md:97]: `If you're using Pterodactyl panel` you probably are if you're reading this section?

File: _docs/geyser/setup.md [hunk 3]
```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
...
-2. Put the Geyser jar file in your plugins folder and start up the server.
-3. If you are on a hosting provider, you will likely need to change your Bedrock port in `config.yml`. Information on your hosting provider might be available on the [Supported Hosting Providers](/geyser/supported-hosting-providers/) page.
-4. Restart your server if you edited your config.
-5. If not using a hosting provider you will need to use port forwarding to allow people out of your LAN to connect. You want to port forward 19132 UDP, instructions to do that for most routers can be found [here](https://www.lifewire.com/how-to-port-forward-4163829).
+1. Read the Prerequisites above, especially if you are setting up geyser for an older Minecraft Java Edition version!
+2. To make sure you are using the easiest way to set up geyser, please check whether your hosting provider features [built-in geyser](/supported-hosting-providers/#built-in-geyser). If you are not using a hosting provider, or your hosting provider is not featured on said list, continue with the steps below.
```
Comment (Camotoy) [inline _docs/geyser/setup.md:35]: capital G Geyser

File: _docs/geyser/setup.md [hunk 4]
```diff
@@ -39,64 +60,31 @@ You can install Floodgate (but not Geyser) on the back-end servers to improve sk
...
-Please keep in mind, you need some sort of computer or host to run Geyser Standalone on. Applications such as Termux on Android are capable of running Geyser, but this largely depends on how powerful your Android device is. Please do so at your own risk. Instructions to run Geyser on Termux can be found [here](/geyser/setup/#termux-android).
-
-1. Download a jar of Geyser from the [build server](https://ci.opencollab.dev/job/Geyser/job/master/).
-2. Create a new folder for Geyser, and drop the jar in there.
-
-### GUI Setup (Recommended)
-3. Double-click the jar file and all the necessary files for Geyser will be created.
-4. Configure options such as the server you want to join in the Geyser config. A description of all options of the config can be found on the [Understanding the Config](/geyser/understanding-the-config/) page.
-5. Stop the current instance of Geyser and re-run it.
+Setup:
```
Comment (Camotoy) [inline _docs/geyser/setup.md:63]: Make this a header?

File: _docs/geyser/setup.md [hunk 5]
```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
...
+ - To check whether it is working correctly, you can use the `geyser connectiontest [yourIP]:[yourPort]` command in your server console to check if the server is reachable from the outside.
+ - _Restart your server if you edited your config! Simply reloading will not work._
+6. Besides the `port` entry in the _Bedrock Section_ of Geyser's `config.yml`, you rarely need to change anything. A few examples:
+ - `address: 0.0.0.0` The IP address that will listen for connections. There is no reason to change this unless you want to limit what IPs can connect to your server.
+ - `port: 19132` The port bedrock players will use to connect. If the option `clone-remote-port` is set to true, this port is ignored, and you'll have to use the same port on bedrock as you would on Java to connect.
+7. The _remote_ section in the `config.yml` determines, which Java server the bedrock players join. 
+- `address: auto` This means, that geyser configures the server's ip, port and auth-type itself. Leaving this at auto is recommended, unless otherwise instructed by a hosting provider.
```
[the remark this answers is not in the document]
Comment (onebeastchris) [reply _docs/geyser/setup.md:47]: I'm assuming you're referring to "address: auto"? In which case, i mostly used the info from here https://wiki.geysermc.org/geyser/understanding-the-config/#remote-section

**Comment by onebeastchris** [comment]
I'm not sure what else to add to the "enforce-secure-profiles" - should there be a warning that chat reporting wouldn't work? Or are the current changes enough

**Comment by Camotoy** [comment]
I would add that other Java players can join that don't abide by chat signing.

**Comment by onebeastchris** [comment]
Something like this? Unfortunately, I'm not entirely sure on the technical process, so i can't give exact info - but considering this is a general setup page, maybe a link to some technical site would be better. But potentially out-of-scope, considering this could make them not want to use Geyser

**Comment by onebeastchris** [comment]
Updated the version from 1.12.2 to 1.13.2 in the Prerequisites section. If you'd like me to change anything, lmk!

**Comment by onebeastchris** [comment]
whoopsie, imma try and fix permalink -

File: _docs/geyser/setup.md [hunk 6]
```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
...
+5. Geyser for Fabric (does not support client required mods, and more info about that can be found [here](/other/geyser-fabric/)),
+6. Geyser Standalone (Standalone Setup instructions are [here](/geyser/standalone/)).
+
+The first five versions run as plugins/mods and can be installed directly onto the server. The standalone version can be used in a similar way, except you run it separately.
 
 If you are running a server, it is highly recommended you use one of the plugin versions, and if you want to join a server that does not have Geyser installed, you can run the standalone version. If you use Pterodactyl Panel we have an egg for the standalone version, please see [here](/geyser/faq/#can-i-use-geyser-with-pterodactyl-panel) for more information.
 
-If you're still having problems with Geyser not working or giving you an "Unable to connect to world" error, see the [Common Issues](/geyser/common-issues/) page.\
-For more information, take a look at the [Understanding the Config](/geyser/understanding-the-config/) page, and the [FAQ](/geyser/faq/) page.\
-And if you still have questions, feel free to join the [Discord](https://discord.geysermc.org) if you haven't already.
```
Comment (jackson-57) [inline _docs/geyser/setup.md:15]: Why was this removed?

File: _docs/geyser/setup.md [hunk 7]
```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
...
 
 ## Prerequisites
-
 - The server you are connecting to has to support the latest version of Minecraft Java Edition (at this time this is Minecraft {{ site.data.versions.java }})
-The server itself does not have to be the latest version but does have to allow connections. If you're running the server on an older version, you can use the plugin [ViaVersion](https://www.spigotmc.org/resources/19254/), but do note that only **1.12.2 and any version above is officially supported**.
+The server itself does not have to be the latest version but does have to allow connections. If you're running the server on an older version, you can use the plugin [ViaVersion](https://www.spigotmc.org/resources/19254/), but do note that only **1.13.2 and any version above is officially supported**. For older versions, use Geyser Standalone, or Geyser on a proxy like Velocity or Bungeecord.
```
Comment (jackson-57) [inline _docs/geyser/setup.md:26]:
```suggestion
The server itself does not have to be the latest version but does have to allow connections. If you're running the server on an older version, you can use the plugin [ViaVersion](https://www.spigotmc.org/resources/19254/), but do note that only **1.13.2 and any version above is officially supported**. For older versions, use Geyser Standalone, or Geyser on a proxy like Velocity or BungeeCord.
```
caps

File: _docs/geyser/setup.md [hunk 8]
```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
...
+6. Besides the `port` entry in the _Bedrock Section_ of Geyser's `config.yml`, you rarely need to change anything. A few examples:
+ - `address: 0.0.0.0` The IP address that will listen for connections. There is no reason to change this unless you want to limit what IPs can connect to your server or if your hosting provider instructs you to do so.
+  - `port: 19132` The port Bedrock players will use to connect. If the option `clone-remote-port` is set to true, this port is ignored, and you'll have to use the same port on bedrock as you would on Java to connect.
+7. The _remote_ section in the `config.yml` determines, which Java server the Bedrock players join. 
+- `address: auto` This means, that Geyser configures the server's ip, port and auth-type of the java server by itself, leave this at auto.
+- `port: 25565` This should be the port of the Java server. By default, it is set to 25565 - your hosting provider may have assigned a different port to your java server, set that here.
+8. If you wish to remove the java account requirement, set up the [Floodgate](/floodgate/setup/) plugin. This will allow Bedrock players to play on your server, without you having to use offline mode.
```
Comment (jackson-57) [inline _docs/geyser/setup.md:49]:
```suggestion
- `address: auto` This means, that Geyser configures the server's IP, port and auth-type of the java server by itself, leave this at auto.
- `port: 25565` This should be the port of the Java server. By default, it is set to 25565 - your hosting provider may have assigned a different port to your Java server, set that here.
8. If you wish to remove the Java account requirement, set up the [Floodgate](/floodgate/setup/) plugin. This will allow Bedrock players to play on your server, without you having to use offline mode.
```

File: _docs/geyser/setup.md [hunk 9]
```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
...
-4. Restart your server if you edited your config.
-5. If not using a hosting provider you will need to use port forwarding to allow people out of your LAN to connect. You want to port forward 19132 UDP, instructions to do that for most routers can be found [here](https://www.lifewire.com/how-to-port-forward-4163829).
+1. Read the Prerequisites above, especially if you are setting up Geyser for an older Minecraft Java Edition version!
+2. To make sure you are using the easiest way to set up Geyser, please check whether your hosting provider features [built-in Geyser](/supported-hosting-providers/#built-in-geyser). If you are not using a hosting provider, or your hosting provider is not featured on said list, continue with the steps below. 
+3. If 2. does not apply: download a jar of Geyser from the [build server](https://ci.opencollab.dev/job/Geyser/job/master/) depending on what platform your server runs on. See the [FAQ](/geyser/faq/#which-plugin-version-of-geyser-do-i-need) if you're confused about which build to download. Keep in mind: Geyser-spigot.jar is for Spigot and it's forks, such as PaperMC or Purpur.
```
Comment (jackson-57) [inline _docs/geyser/setup.md:36]:
```suggestion
3. If 2. does not apply: download a jar of Geyser from the [build server](https://ci.opencollab.dev/job/Geyser/job/master/) depending on what platform your server runs on. See the [FAQ](/geyser/faq/#which-plugin-version-of-geyser-do-i-need) if you're confused about which build to download. Keep in mind: Geyser-Spigot.jar is for Spigot and its forks, such as PaperMC or Purpur. 
```
caps, incorrect punctuation

File: _docs/geyser/setup.md [hunk 10]
```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
...
 - If you are connecting to an online mode Java server, a paid Java account is required. If you are running the server, you can bypass this requirement for your server with [Floodgate](/floodgate/).
 - Your Bedrock client has to be a supported version - at this time that is Bedrock version(s) {{ site.data.versions.bedrock }}.
 - If you are running the server, you need to have a UDP port opened. See below for more instructions.
+- Currently, Geyser does not support key signing. To disable it (on servers running version 1.19 and higher), follow these instructions: Spigot, Paper, & all forks: Set `enfore-secure-profile: false` in the `server.properties` file. On BungeeCord: Set `enforce_secure_profile: false` in your `config.yml`. Using Velocity: set `force-key-authentication = false` in your `velocity.toml`. This will disable chat signing, which Geyser currently does not support. Be careful though: This will allow java players who bypass chat signing with mods such as NoChatReports to join the server, and make them unreportable. For more info on chat reporting, see this [article](https://gist.github.com/kennytv/ed783dd244ca0321bbd882c347892874).
```
Comment (jackson-57) [inline _docs/geyser/setup.md:31]:
```suggestion
- Currently, Geyser does not support key signing. To disable it (on servers running version 1.19 and higher), follow these instructions: Spigot, Paper, & all forks: Set `enfore-secure-profile: false` in the `server.properties` file. On BungeeCord: Set `enforce_secure_profile: false` in your `config.yml`. Using Velocity: set `force-key-authentication = false` in your `velocity.toml`. This will disable chat signing, which Geyser currently does not support. Be careful though: This will allow Java players who bypass chat signing with mods such as NoChatReports to join the server, and make them unreportable. For more info on chat reporting, see this [article](https://gist.github.com/kennytv/ed783dd244ca0321bbd882c347892874).
```
caps

File: _docs/geyser/setup.md [hunk 11]
```diff
@@ -39,64 +60,31 @@ You can install Floodgate (but not Geyser) on the back-end servers to improve sk
...
+2. Download a jar of Geyser from the [build server](https://ci.opencollab.dev/job/Geyser/job/master/) depending on what platform your server runs on. See the [FAQ](/geyser/faq/#which-plugin-version-of-geyser-do-i-need) if you're confused about which build to download. 
+3. Put the Geyser jar file in your plugins folder and start up the server.
+4. Port-forwarding:
+- If you are using a hosting provider, you will likely need to change your Bedrock port in the `config.yml` file. Information on your hosting provider might be available on the [Supported Hosting Providers](/geyser/supported-hosting-providers/) page. If your hosting provider is not listed there, and you are unable to port-forward the default 19132 port on UDP, try setting "clone-remote-port" to true and connecting with the java ip and port. Any port will work, as long as the protocol UDP is not blocked or already in use by the query feature/another mod.
```
Comment (jackson-57) [inline _docs/geyser/setup.md:68]:
```suggestion
- If you are using a hosting provider, you will likely need to change your Bedrock port in the `config.yml` file. Information on your hosting provider might be available on the [Supported Hosting Providers](/geyser/supported-hosting-providers/) page. If your hosting provider is not listed there, and you are unable to port forward the default 19132 port on UDP, try setting `clone-remote-port` to true and connecting with the Java IP and port. Any port will work, as long as the UDP protocol is not blocked or already in use by the query feature (see `server.properties`) or another mod. 
```
caps, punctuation, formatting

File: _docs/geyser/setup.md [hunk 12]
```diff
@@ -39,64 +60,31 @@ You can install Floodgate (but not Geyser) on the back-end servers to improve sk
...
+- To check whether it is working correctly, you can use the `geyser connectiontest [yourIP]:[yourPort]` command in your server console to check if the server is reachable from the outside.
+-  _Restart your server if you edited your config! Simply reloading will not work._
+5. Besides the `port` entry in the _Bedrock Section_ of Geyser's `config.yml`, you rarely need to change anything. A few examples:
+- `address: 0.0.0.0` The IP address that will listen for connections. There is no reason to change this unless you want to limit what IPs can connect to your server or if your hosting provider instructs you to do so.
+- `port: 19132` The port bedrock players will use to connect. If the option `clone-remote-port` is set to true, this port is ignored, and you'll have to use the same port on Bedrock as you would on Java to connect.
+6. The _remote_ section in the `config.yml` determines, which Java proxy the Bedrock players join. The proxy then configures, which server the players are sent to.
```
Comment (jackson-57) [inline _docs/geyser/setup.md:75]:
```suggestion
6. The _remote_ section in the `config.yml` determines which Java proxy the Bedrock players join. The proxy then configures which server the players are sent to.
```
extra commas?

**Comment by onebeastchris** [comment]
should hopefully work now, i forgot to change the page permalink

File: _docs/geyser/setup.md [hunk 13]
```diff
@@ -39,64 +60,31 @@ You can install Floodgate (but not Geyser) on the back-end servers to improve sk
...
+- `address: auto` This means, that Geyser configures the proxy's ip, port and auth-type itself. Leaving this at auto is recommended.
```
Comment (jackson-57) [inline _docs/geyser/setup.md:76]:
```suggestion
- `address: auto` Leaving this at auto is recommended, because it means that Geyser configures the proxy's IP, port and auth-type itself.
```
reworded, feel free to reject

File: _docs/geyser/standalone.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+title: Geyser Setup
+permalink: /geyser/setup/
+---
+
+# Setup of Geyser Standalone
+
+
+## Prerequisites
+
+- The server you are connecting to has to support the latest version of Minecraft Java Edition (at this time this is Minecraft {{ site.data.versions.java }})
```
Comment (jackson-57) [inline _docs/geyser/standalone.md:12]:
```suggestion
- The server you are connecting to has to support the latest version of Minecraft: Java Edition (at this time this is Minecraft {{ site.data.versions.java }})
```
proper title

File: _docs/geyser/standalone.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+The server itself does not have to be the latest version but does have to allow connections. If you're running the server on an older version, you can use the plugin [ViaVersion](https://www.spigotmc.org/resources/19254/).
+- The device running Geyser must support Java 16 or later. If you need help installing or updating, please see [PaperMC's Java update page](https://paper.readthedocs.io/en/latest/java-update/index.html).
+- If you are connecting to an online mode Java server, a paid Java account is required. If you are running the server, you can bypass this requirement for your server with [Floodgate](/floodgate/).
+- Your Bedrock client has to be a supported version - at this time that is Bedrock version(s) {{ site.data.versions.bedrock }}.
+- To allow Geyser Standalone to be reached outside the local network, Unlike Minecraft Java Edition, Bedrock Edition runs on port 19132 on the UDP protocol. When port forwarding, make sure to allocate to 19132 UDP or another UDP port.
```
Comment (jackson-57) [inline _docs/geyser/standalone.md:17]:
```suggestion
- To allow Geyser Standalone to be reached outside the local network, Unlike Minecraft: Java Edition, Bedrock Edition runs on port 19132 on the UDP protocol. When port forwarding, make sure to allocate to 19132 UDP or another UDP port.
```
title

File: _docs/geyser/standalone.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+Once you're done, open up Minecraft: Bedrock Edition and in the **Friends** tab, Geyser should show up there. If it does not show up, just add the Java IPv4 address and Bedrock port.
+
+### Standalone Setup (Geyser and Floodgate are not on the same server)
+If you want to offload Geyser's processing you can host Geyser (Standalone) somewhere else and make it redirect to your server.
+**Warning:** Do not forget to edit the remote section of your Geyser config.
+See the [Floodgate](/floodgate/) wiki page for more details.
+
+## Port Forwarding
+
+Unlike Minecraft Java Edition, Bedrock Edition runs on port 19132 on the UDP protocol. When port forwarding, make sure to allocate to 19132 UDP or another UDP port. For many server hosting providers, you will simply need to change your Bedrock listening port (see [here](/geyser/supported-hosting-providers/) for a list of supported providers).
```
Comment (jackson-57) [inline _docs/geyser/standalone.md:47]:
```suggestion
Unlike Minecraft: Java Edition, Bedrock Edition runs on port 19132 on the UDP protocol. When port forwarding, make sure to allocate to 19132 UDP or another UDP port. For many server hosting providers, you will simply need to change your Bedrock listening port (see [here](/geyser/supported-hosting-providers/) for a list of supported providers).
```
title

File: _docs/geyser/standalone.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+
+## NewTerm 2 (iOS)
+**Note:** A jailbreak is required. You can find what jailbreak to use for your device [here](https://docs.google.com/spreadsheets/d/11DABHIIqwYQKj1L83AK9ywk_hYMjEkcaxpIg6phbTf0/edit?usp=sharing).
+1. Install [Filza File Manager](http://cydia.saurik.com/package/com.tigisoftware.filza/).
+2. Install [NewTerm 2](https://chariz.com/get/newterm).
+4. Download jre-16 for iOS and install it using Filza [here](https://github.com/PojavLauncherTeam/PojavLauncher_iOS/releases/download/v16-openjdk/openjdk-16-jre_16.0.0+git20201217.8383f41-2_iphoneos-arm.deb).
+3. Download my modified java commands, and install it using Filza [here](https://cdn.discordapp.com/attachments/558829512633090048/834014323755319306/com.letschill.java_0.1_iphoneos-arm.deb).
+4. Open NewTerm 2 and run `wget https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/standalone/build/libs/Geyser-Standalone.jar`
+5. Run `java -jar Geyser-Standalone.jar`.
+6. You can find Geyser server running in MCPE!
```
Comment (jackson-57) [inline _docs/geyser/standalone.md:72]:
```suggestion
4. Download PojavLauncher's Java 16 JRE for iOS and install it using Filza [here](https://github.com/PojavLauncherTeam/PojavLauncher_iOS/releases/download/v16-openjdk/openjdk-16-jre_16.0.0+git20201217.8383f41-2_iphoneos-arm.deb).
3. Download this package containing modified java commands, and install it using Filza [here](https://cdn.discordapp.com/attachments/558829512633090048/834014323755319306/com.letschill.java_0.1_iphoneos-arm.deb) (Note: Not an official GeyserMC project, install at your own risk).
4. Open NewTerm 2 and run `wget https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/standalone/build/libs/Geyser-Standalone.jar`
5. Run `java -jar Geyser-Standalone.jar`.
6. Geyser should show up in the Friends tab inside Minecraft.
```
clarified origins of files, made more specific

File: _docs/geyser/standalone.md [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+
+**Note:**
+Due to iOS's environment, if your device has low specs, iOS might end up killing NewTerm 2 while you're playing, stopping the server. There is no fix for that because of how iOS works.
+
+You may encounter some issues. If you do, run `su` then enter the root password (default is `alpine`) for root access. Then, run the server like you normally would, and it should work just fine.
```
Comment (jackson-57) [inline _docs/geyser/standalone.md:77]: This is probably an awful idea, security wise. Should we remove this section, or at least add a disclaimer?

File: _docs/geyser/standalone.md [hunk 19]
```diff
@@ -0,0 +1,78 @@
...
+1. Install [Filza File Manager](http://cydia.saurik.com/package/com.tigisoftware.filza/).
+2. Install [NewTerm 2](https://chariz.com/get/newterm).
+4. Download jre-16 for iOS and install it using Filza [here](https://github.com/PojavLauncherTeam/PojavLauncher_iOS/releases/download/v16-openjdk/openjdk-16-jre_16.0.0+git20201217.8383f41-2_iphoneos-arm.deb).
+3. Download my modified java commands, and install it using Filza [here](https://cdn.discordapp.com/attachments/558829512633090048/834014323755319306/com.letschill.java_0.1_iphoneos-arm.deb).
+4. Open NewTerm 2 and run `wget https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/standalone/build/libs/Geyser-Standalone.jar`
+5. Run `java -jar Geyser-Standalone.jar`.
+6. You can find Geyser server running in MCPE!
+
+**Note:**
+Due to iOS's environment, if your device has low specs, iOS might end up killing NewTerm 2 while you're playing, stopping the server. There is no fix for that because of how iOS works.
```
Comment (jackson-57) [inline _docs/geyser/standalone.md:75]:
```suggestion
Due to the iOS environment, if your device has low specs, iOS might end up killing NewTerm 2 while you're playing, stopping the server. There is no fix for that because of how iOS works.
```
grammar

File: _docs/geyser/setup.md [hunk 20]
```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
...
+3. If 2. does not apply: download a jar of Geyser from the [build server](https://ci.opencollab.dev/job/Geyser/job/master/) depending on what platform your server runs on. See the [FAQ](/geyser/faq/#which-plugin-version-of-geyser-do-i-need) if you're confused about which build to download. Keep in mind: Geyser-spigot.jar is for Spigot and it's forks, such as PaperMC or Purpur. 
+4. Put the Geyser jar file in your plugins folder and start up the server.
+5. Port-forwarding: 
+ - If you are using a hosting provider, you will likely need to change your Bedrock port in `config.yml`. Information on your hosting provider might be available on the [Supported Hosting Providers](/geyser/supported-hosting-providers/) page. If there is no information on your hosting provider and you are unable to assign/open ports yourself , try enabling `clone-remote-port` in Geyser's `config.yml`file and connecting with the same IP/port as you would with Java Edition.
```
Comment (jackson-57) [inline _docs/geyser/setup.md:39]:
```suggestion
 - If you are using a hosting provider, you will likely need to change your Bedrock port in `config.yml`. Information on your hosting provider might be available on the [Supported Hosting Providers](/geyser/supported-hosting-providers/) page. If there is no information on your hosting provider and you are unable to assign/open ports yourself, try enabling `clone-remote-port` in Geyser's `config.yml`file and connecting with the same IP/port as you would with Java Edition.
```
extra space

**Review by jackson-57 — CHANGES_REQUESTED** [review]
I went through and left suggestions for grammar and punctuation. Most of it is quite minor. If you disagree with anything, feel free to reject the suggestion. I wasn't super thorough in my review, so I might have missed something.

File: _docs/geyser/setup.md (hunk 6 above)
Comment (onebeastchris) [reply _docs/geyser/setup.md:15]: It's not removed, i just moved it to another section - i thought it would make more sense to have this after the setup guide, not infront of it

Comment (jackson-57) [reply _docs/geyser/setup.md:15]: I meant to delete this comment, but it disappeared from the review. Thanks GitHub!

File: _docs/geyser/standalone.md (hunk 17 above)
Comment (onebeastchris) [reply _docs/geyser/standalone.md:72]: yea i didn't actually change any of the standalone instructions, just moved them to a separate file to de-clutter the plugin setup page. I'll just add this, definitely won't hurt

File: _docs/geyser/standalone.md (hunk 18 above)
Comment (onebeastchris) [reply _docs/geyser/standalone.md:77]: No idea honestly - I'd remove it tbh

File: _docs/geyser/standalone.md (hunk 17 above)
Comment (jackson-57) [reply _docs/geyser/standalone.md:72]: I thought that we might as well clean this section up, given the opportunity.

**Review by jackson-57 — APPROVED** [review]

**Comment by onebeastchris** [comment]
now to fix the missing permalink in the toc.yml file - I'll try and fix that

**Comment by onebeastchris** [comment]
okay, should be fixed now. I'd try it myself, but for some reason, getting the website to run has proven to be a bit.. difficult. Will try later, but I'd say there's a decent chance it'll work

**Review by Jens-Co — APPROVED** [review]
Thanks a lot for this pr, nice work!

**Review by Bijoygamer — COMMENTED** [review]
O

**Comment by Chew** [comment]
Please don't necro merged PRs from months ago, seriously man?
